robots.txt

```diff
@@ -1,5 +1,3 @@
 ---
 ---
 User-agent: *
-{% for version in site.versions %}
-Disallow: /docs/envoy/{{ version }}/{% endfor %}
```

Comment:
I just realized also that this was broken as there is no `v` prefixing the version. But this solution is still the correct one.

Reply:
cc @lucperkins

Reply:
Ah yes, an oversight on my part.